### PR TITLE
Ry4

### PR DESCRIPTION
**Pull request by ryhorowitz**
## Description
Map skeleton. Just a map with a marker. 

If anyone can help me with Google API calls that would be most appreciated.  It's been 5 days now and I got nothing.

### CODE REVIEW

**Review by Laweeza — APPROVED** [review]
Skeleton looks okay for component

**Comment by Laweeza** [comment]
[Map Skeleton](https://trello.com/c/ijYO33lC/45-map-skeleton)

File: .gitignore [hunk 1]
```diff
@@ -10,6 +10,6 @@ npm-debug.*
 *.mobileprovision
 *.orig.*
 web-build/
-
+app.json
```
Comment (shabbyblue16) [inline .gitignore:15]: Good thought with the API keys. We'll have to make sure everybody knows to recreate app.json file locally.

File: client/components/Map/Map.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+import userLocationState from '../../state/atoms/userLocation';
+import currentMapView from '../../state/atoms/currentMapView';
+const { GOOGLE_MAPS_API_KEY_IOS } = process.env;
+
+const { height, width } = Dimensions.get('window');
+
+const Map = () => {
+  const currentView = useRecoilValue(currentMapView);
+  const userLocation = useRecoilValue(userLocationState);
```
Comment (shabbyblue16) [inline client/components/Map/Map.jsx:22]: I think to use this atom `useRecoilState(currentMapView)` is needed, but if this works then no big deal.

**Review by shabbyblue16 — APPROVED** [review]
Good stuff, Google Maps API is a beast. My component's pretty simple so I'm happy to help if you need to delegate.

**Pull request merged**